docs/_posts/0000-03-01-object-orientation.md

```diff
@@ -128,3 +140,38 @@ Note: Get them to describe their designs.
 
 Note:
 Needs a better diagram, but the point is here that these concepts are not distinct, there is often quite a lot of overlap between these language features.  
+
+--
+
+## Discussion: What is Object Orientation trying to achieve?
+
+* What is hard to do without OOP?
+* What are the potential benefits?
+* If we didn’t use OOP, what might happen to our code?
+* In short: what is the point?
+
+Note: Could use Miro or Slido for this, and could be in breakout rooms if we want discussion or just solo if they add stickies to Miro 
```

Comment:
Not sure we need a formal breakout discussion, maybe just ask for some ideas (think they might struggle with such a broad question)

Reply:
Yep, happy with that. I think we should go with a Miro board that they contribute then, since we will be using Miro for other bits of the session. ~Five minutes for them to write stickies?